src/main/java/com/hackerrank/weather/controller/WeatherApiRestController.java

```diff
@@ -54,28 +55,40 @@ public List<Weather> weather(@RequestParam("date") String date) {
     }
 
     @GetMapping("/city/{cityName}")
-    public ArrayList<Double> getWeatherTemperatureByCityName(@PathVariable(name = "cityName") String cityName){
+    public ArrayList<Double> getWeatherTemperatureByCityName(@PathVariable(name = "cityName") String cityName) {
         return weatherService.getWeatherTemperatureByCityName(cityName);
     }
 
     @DeleteMapping("/delete/id/{id}")
-    public void deleteWeatherById(@PathVariable(name = "id") Integer id){
+    public void deleteWeatherById(@PathVariable(name = "id") Integer id) {
         weatherService.deleteWeatherById(id);
     }
 
     @DeleteMapping("/delete/name/{cityName}")
-    public void deleteFirstInstanceOfWeatherByCity(@PathVariable(name = "cityName")String cityName){
+    public void deleteFirstInstanceOfWeatherByCity(@PathVariable(name = "cityName") String cityName) {
```

Comment:
Please, Add @ApiOperation with all properties that make sense for that endpoint.